activity/activity.info

```diff
@@ -1,8 +1,6 @@
 [Activity]
-name = falabracman 
-activity_version = 2
-host_version = 1
-bundle_id = org.laptop.community.falabracman
+name = SugargameTest
```

Comment:
The name and bundle_id aren't supposed to change in this case.

Reply:
I will change that once I get it working somehow, I used the template